### ISSUE TITLE
Review 1.6 updates

### ISSUE THREAD

**Issue by ryanrichholt** [tgen/jetstream#92]
I've got most of the changes committed and moved to a release branch now. Looking for help testing the latest version. 

Things that I need help reviewing in the feature branch before merging to master:

- [x] `reset` directive
- [x] `--pipeline` option with render/build commands
- [x] pipeline/project environment variables
- [x] config file inputs with -C/-c  new type options are available
- [x] Documents

**Comment by ryanrichholt** [comment]
This update can be installed with:

```
 pip install --upgrade git+https://github.com/tgen/jetstream.git@release-1.6
```

**Comment by awchrist** [comment]
Running into an error while using the above command. It looks like it is trying to install jetstream into /packages and failing due to permissions. The error log is a bit long.

**Comment by TGEN-BTurner** [comment]
I was able to get through install using 
```pip3 install --user --upgrade git+https://github.com/tgen/jetstream.git@release-1.6```
However, the jetstream_centro run_pipeline breaks when trying to run the retrieve_archive task with the following error ```sbatch: error: Unable to open file retrieve_project.sh```

**Comment by ryanrichholt** [comment]
@awchrist If you're not running in a virtualenv or using root privileges then you'll need to add the `--user` flag to the command.

**Comment by ryanrichholt** [comment]
@TGEN-BTurner Do you think the problem could be v1.5 was adding `retrieve_project.sh` to the PATH, but v1.6 is not? It looks like that script is in a scripts directory, I'll review the changes to see if that directory should have been added to the PATH.

**Comment by awchrist** [comment]
Thanks Ryan. I think we forgot to restart the server from within the virtual env. Don't worry about looking until we test that out.

**Comment by awchrist** [comment]
@ryanrichholt Ok I can confirm the error @TGEN-BTurner was getting with `retrieve_project.sh` is specific to jetstream v1.6.

**Comment by awchrist** [comment]
@ryanrichholt I believe that it is using the "bin:" key in the yaml to add the path.

**Comment by ryanrichholt** [comment]
@awchrist Thanks, that helped me figure it out. I forgot that it was setup in the yaml, and was only adding it if a literal "bin" directory existed. Fixed in a recent commit, Im working on resolving other test failures with the new Github workflow and then I'll PR to this branch.

**Comment by ryanrichholt** [comment]
Recent changes are merged into release-1.6 and passing tests
